bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs/diff/ObsNumberCorrection.scala

```diff
@@ -25,15 +26,16 @@ import Scalaz._
   * remote program versions with an observation number greater than a
   * local-only observation.
   */
-class ObsNumberCorrection(isKnown: (ProgramLocation, SPNodeKey) => Boolean) extends CorrectionFunction {
+class ObsNumberCorrection(lifespanId: LifespanId, isKnown: (ProgramLocation, SPNodeKey) => Boolean) extends CorrectionFunction {
   def apply(mp: MergePlan): TryCorrect[MergePlan] =
     renumberedObs(mp).map { obsMap =>
-      if (obsMap.isEmpty)  // usually empty so we might as well check and save a traversal in that case
-        mp
+      if (obsMap.isEmpty) mp // usually empty so we might as well check and save a traversal in that case
```

Comment:
The traversal refers to mapping over the whole `Tree[MergeNode]` contained in the plan.  If there are no renumbered observations, and usually there won't be, it skips the map.